artifactory/commands/transferfiles/utils.go

```diff
@@ -264,13 +267,22 @@ func getErrorsFiles(repoKey string, isRetry bool) (filesPaths []string, err erro
 		return []string{}, err
 	}
 
-	filesNames, err := biUtils.ListFiles(dirPath, false)
+	errorsFilesRegex := fmt.Sprintf(errorsFilesRegexFormat, repoKey)
+
+	regExp, err := regexp.Compile(errorsFilesRegex)
```

Comment:
```regexp.Compile``` is an expensive operation. It is therefore better to it only once per CLI execution.

Reply:
This regexp contains the repo key, so it cannot be compiled once. As discussed, leaving as it is